WordPress/Classes/Services/MediaRepository.swift

```diff
@@ -59,4 +59,31 @@ final class MediaRepository {
 
         return nil
     }
+
+}
+
+extension MediaServiceRemote {
+
+    func getMediaLibraryCount(forMediaTypes types: [MediaType]) async throws -> Int {
```

Comment:
Oh nice, I didn't know it existed. We may be able to implement the post/pages/media counters in the new dashboard menu after all.